theme/404.html

```diff
@@ -57,18 +92,17 @@ <h1 class="no-auto-render">
                                 <span class="header-badge custom-badge" style="background-color: #CC334A; ">
                                     <i class="fa fa-exclamation-triangle"></i>
                                 </span>
-                                Sorry the Page Cannot be Found!</h1>
+                                Sorry, the Page Cannot Be Found!</h1>
                             <p style="height: 30rem">
-                                Sorry, we couldn't find the page you were looking for. If you believe this is an error please contact us at
-                                <a href="mailto:dcc-support@icgc.org?subject=ICGC%20DCC%20Document%20Page%20Not%20Found">dcc-support@icgc.org</a>.
```

Comment:
No more contact us option?

Reply:
Good point. I'll leave it up to @wwysoc2 as to whether it'd help to have a contact address on our 404 page. It shouldn't be this address, though.

Reply:
I put the back and updated the address to `support@nci-gdc.datacommons.io`.